Objects/memoryobject.c

```diff
@@ -2332,7 +2332,9 @@ ptr_from_tuple(Py_buffer *view, PyObject *tup)
 
 /* Return the item at index. In a one-dimensional view, this is an object
    with the type specified by view->format. Otherwise, the item is a sub-view.
-   The function is used in memory_subscript() and memory_as_sequence. */
+   The function is used by memory_subscript(), memoryiter_next() and
+   memory_as_sequence.
+   Note: Iteration of multi-dimensional arrays is not yet implemented */
```

Comment:
I think you can revert this whole comment, you no longer use `memory_item` in `memoryiter_next`.

Reply:
Fixed.